### PR TITLE
Improve Perf System

### PR DESCRIPTION
**Pull request by oliviajain**
**Description**: 
- refactor table names
- load TRT inputs
- Split up anubis job and perf script

**Motivation and Context**
- Increase maintainability of perf script.

### CODE REVIEW

File: tools/ci_build/github/azure-pipelines/linux-gpu-tensorrt-daily-perf-pipeline.yml [hunk 1]
```diff
@@ -93,6 +93,7 @@ jobs:
       value: $(environment) -e "$(epList)"
 
   steps:
+    - script: echo "https://aiinfra.visualstudio.com/Lotus/_build/results?buildId=$(Build.BuildId)"
```
Comment (jywu-msft) [inline tools/ci_build/github/azure-pipelines/linux-gpu-tensorrt-daily-perf-pipeline.yml:96]: why do we need to embed urls in the yml file?

File: tools/ci_build/github/azure-pipelines/linux-gpu-tensorrt-daily-perf-pipeline.yml [hunk 2]
```diff
@@ -172,7 +173,7 @@ jobs:
           scriptType: bash
           inlineScript: |
             short_hash=$(git rev-parse --short HEAD^) && 
-            python3 $(Build.SourcesDirectory)/onnxruntime/python/tools/tensorrt/perf/post.py -r $(Build.SourcesDirectory)/Artifact/result -c $short_hash -u "https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=$(Build.BuildId)" -t ${{ parameters.TrtVersion }} -b $(branch)
+            python3 $(Build.SourcesDirectory)/onnxruntime/python/tools/tensorrt/perf/post.py -r $(Build.SourcesDirectory)/Artifact/result -c $short_hash -u "https://aiinfra.visualstudio.com/Lotus/_build/results?buildId=$(Build.BuildId)" -t ${{ parameters.TrtVersion }} -b $(branch)
```
Comment (jywu-msft) [inline tools/ci_build/github/azure-pipelines/linux-gpu-tensorrt-daily-perf-pipeline.yml:176]: same comment. why is this needed?

Comment (oliviajain) [reply tools/ci_build/github/azure-pipelines/linux-gpu-tensorrt-daily-perf-pipeline.yml]: Changed to: 

"$(reportUrl)?buildId=$(Build.BuildId)"

**Review by jywu-msft — APPROVED** [review]
